potatolondon/djangae
# Added real documentation OMG! 🔥

Sooo, I really didn't want to write my talk tonight so to procrastinate a little bit I decided to make this. This PR should resolve issue #326 and basically all of our problems (obvs)

To test locally:

1. `pip install mkdocs`
2. `cd djangae`
3. `mkdocs serve`
4. Go to http://127.0.0.1:8000/

Known issues & questions:

1. I think mkdocs has a slightly different ReadTheDocs theme, hopefully this will update when we push it to RTD
2. I'm not sure if "Unique constraint tool" should be its own chapter, or maybe go into the "Unique Constraint Checking" chapter
3. I'm not sure if "Unique Constraint Checking" should be its own chapter, or maybe go into the "Database Backend" chapter
4. Does the order of chapters even make sense?

Yay! :tada:
